hoodie-hive/src/main/java/com/uber/hoodie/hive/HoodieHiveClient.java

```diff
@@ -152,45 +153,41 @@ private String constructAddPartitions(List<String> partitions) {
     alterSQL.append(syncConfig.databaseName).append(".").append(syncConfig.tableName)
         .append(" ADD IF NOT EXISTS ");
     for (String partition : partitions) {
-
-      StringBuilder partBuilder = new StringBuilder();
-      List<String> partitionValues = partitionValueExtractor
-          .extractPartitionValuesInPath(partition);
-      Preconditions.checkArgument(syncConfig.partitionFields.size() == partitionValues.size(),
-          "Partition key parts " + syncConfig.partitionFields
-              + " does not match with partition values " + partitionValues
-              + ". Check partition strategy. ");
-      for (int i = 0; i < syncConfig.partitionFields.size(); i++) {
-        partBuilder.append(syncConfig.partitionFields.get(i)).append("=").append("'")
-            .append(partitionValues.get(i)).append("'");
-      }
-
+      String partitionSpec = getPartition(partition);
```

Comment:
"partitionSpec" is confusing, could we rename this to partitionClause ?

Reply:
Done.